### PR TITLE
Added plugin toolbar

### PR DESCRIPTION
**Pull request by erpas**

### CODE REVIEW

**Comment by erpas** [comment]
Plugin toolbar screenshot:

mergin_plugin_toolbar

File: Mergin/plugin.py [hunk 1]
```diff
@@ -623,6 +703,21 @@ def actions(self, parent):
         return actions
 
 
+class MerginProjectsManager(object):
```
Comment (DamnBack) [inline Mergin/plugin.py:706]: Could you add some docstrings with brief class description?

File: Mergin/plugin.py [hunk 2]
```diff
@@ -76,8 +82,68 @@ def initGui(self):
...
+    def enable_toolbar_actions(self, enable=True):
+        for action in self.toolbar.actions():
+            if action.text() == "Configure Mergin Plugin":
+                continue
+            action.setEnabled(enable)
+
+    def configure(self):
+        self.data_item_provider.root_item.configure()
+
+    def cur_project_status(self):
```
Comment (DamnBack) [inline Mergin/plugin.py:138]: Can we have "current" instead of just "cur" in method name?

File: Mergin/plugin.py [hunk 3]
```diff
@@ -76,8 +82,68 @@ def initGui(self):
...
+        self.data_item_provider.root_item.manager.status(self.mergin_proj_name)
+
+    def cur_project_sync(self):
```
Comment (DamnBack) [inline Mergin/plugin.py:141]: Can we have "current" instead of just "cur" in method name?

File: Mergin/plugin.py (hunk 1 above)
Comment (erpas) [reply Mergin/plugin.py:706]: Done

**Comment by DamnBack** [comment]
Looks good to me now. @wonder-sk feel free to take it over. :)

**Pull request merged**